src/disc.py

```diff
@@ -123,6 +123,17 @@ def dust_frac(self):
     def amax(self):
         return 0
 
+    def new_Omega_k(self, R):
```

Comment:
Do we still need this? I thought this was something used in debugging but that we no longer need?

Reply:
True, we don't - removed